app/views/content/presentations.md

```diff
@@ -43,14 +37,3 @@ This presentation covers:
 - Advice and support from teacher training advisers
 
 $train-to-teach-and-support$
-
-## Ready to Apply?
```

Comment:
The intro paragraph on this page doesn't make sense any more - "There are two presentations"

I will contact the events team about removing these presentations altogether from the site.

Reply:
Any update on this from the events team?